README.md

```diff
@@ -123,6 +123,24 @@ Handler params are:
 - workflow instance
 - payload - None
 
+
+### Support for custom data
+Both `Workflow`and `WorkflowNode` have a `user_params` dictionary member that can be used to store 
```

Comment:
Would you mind if I call it ```custom_payload```?
```user_params``` indicates that it should store some user parameters, which brings more questions - e.g. what kind of params? At least that's my reasoning.

Reply:
oh yeah that is definetely a much better name !